modules/appeals_api/app/controllers/appeals_api/application_controller.rb

```diff
@@ -3,5 +3,40 @@
 module AppealsApi
   class ApplicationController < ::ApplicationController
     skip_before_action :set_tags_and_extra_context
+    before_action :log_request
+
+    private
+
+    def log_request
+      hashed_ssn = Digest::SHA2.hexdigest ssn
+      Rails.logger.info('Caseflow Request',
```

Comment:
If this is going to be used for both appeals and claims it should probably have a different name than 'Caseflow' which is the name of the Appeals Rails app.